github-profiles/Marcos.-0222/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Profile page</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</html>
+<head>
+<h1>🖐🏻Hello, I'm Marcos Viejo 🖐🏻
+    </h1>
+    </head>
+<body>
+    <h2>This is a briefly description about me</h2>
+    <h2>I'm a <strong>lawyer</strong> and working as <strong>Productivy Manager</strong> at the present
+        </h2>
+    <p1>I always love computing and I like to learn by myself a lot of things related to this computing's world.
```

Comment:
`p` tags do not use numerals, they are not p1, p2.... just `<p></p>` and you can write as many paragraph as you need

Reply:
Hi Alianna,
Thank you so much for your comments. I really appreciate you having some time to review it. I will definitely take a look at them and make improvements in my files.
Thank you again!